README.md

```diff
@@ -142,6 +142,16 @@ See the [documentation](Documentation/ConsumeNightlyBuild.md)
 
 If you find a bug or have a feature request, please report them at this repository's issues section. See the [CONTRIBUTING GUIDE](CONTRIBUTING.md) for details on building and contributing to this project.
 
+## Coverlet Team
+
+Author and owner    
+* [Toni Solarin-Sodara](https://github.com/tonerdo)  
+
+Co-maintainers
+
+* [Peter Liljenberg](https://github.com/petli)  
+* [Marco Rossignoli](https://github.com/MarcoRossignoli)
```

Comment:
Put your name on top of mine, you do way more here!

Reply:
Alphabetical order by surname!